frontend/public/components/resource-quota.jsx

```diff
@@ -73,9 +73,9 @@ const getResourceUsage = (quota, resourceType) => {
 };
 
 const tableColumnClasses = [
-  classNames('col-md-5', 'col-xs-6'),
-  classNames('col-md-7', 'col-xs-6'),
-  Kebab.columnClass,
+  'pf-u-w-42-on-lg',
+  'pf-u-w-58-on-lg',
+  `${Kebab.columnClass} pull-right`,
```

Comment:
Similar here. `pull-right` is only required because the namespace column is not shown when a namespace is selected. And the other one has a fix width. So the kebab column will take over the rest of the space.

This means on a ~ 1100px browser window, most space is wasted for the action menu and we have less then 300px for the name column. Which is many cases more then enough. But I personally would prefer a 'dynamic column width'  instead of a hardcoded 5:7 (name to namespace) column width, which makes problem when the namespace is now shown. Wdyt?

![image](https://user-images.githubusercontent.com/139310/123614965-227d3880-d805-11eb-9476-ac11fcd47a4b.png)

```suggestion
  '',
  '',
  Kebab.columnClass,
```

Reply:
updated